### PR TITLE
Update analyzer package

### PR DESCRIPTION
**Pull request by annelo-msft**
New analyzer package includes namespace for Cognitive.

### CODE REVIEW

**Review by pakrym — CHANGES_REQUESTED** [review]
This PR should've ran all the pipelines but it only ran management which doesn't even use this package.

**Comment by annelo-msft** [comment]
@pakrym Does this look like the right set of pipelines?

File: sdk/core/ci.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ trigger:
   paths:
     include:
     - sdk/core/
+    - eng/
```
Comment (pakrym) [inline sdk/core/ci.yml]: Shouldn't this be added to both PR and Trigger sections.

Comment (weshaggard) [reply sdk/core/ci.yml]: Yes it would be good to have it in both.

**Comment by pakrym** [comment]
Do we have any `#pragma warning`s in code that to workaround the issue with analyzer that's being fixed.

File: sdk/core/ci.yml (hunk 1 above)
Comment (annelo-msft) [reply sdk/core/ci.yml]: Added.

**Comment by pakrym** [comment]
We do: https://github.com/Azure/azure-sdk-for-net/search?q=%22namespace+Azure.AI%22&unscoped_q=%22namespace+Azure.AI%22

**Review by weshaggard — APPROVED** [review]

**Review by pakrym — APPROVED** [review]

**Pull request merged**